mlos_bench/mlos_bench/schedulers/base_scheduler.py

```diff
@@ -133,9 +145,28 @@ def experiment(self) -> Storage.Experiment | None:
         return self._experiment
 
     @property
-    def environment(self) -> Environment:
-        """Gets the Experiment Storage."""
-        return self._environment
+    def root_environment(self) -> Environment:
+        """
+        Gets the root (prototypical) Environment from the first TrialRunner.
+
+        Notes
+        -----
+        All TrialRunners have the same Environment config and are made
+        unique by their use of the unique trial_runner_id assigned to each
+        TrialRunner's Environment's global_config.
+        """
+        # Use the first TrialRunner's Environment as the root Environment.
+        return self._trial_runners[self._trial_runner_ids[0]].environment
```

Comment:
maybe just keep the root env runner id?
```suggestion
        return self._trial_runners[self._root_trial_runner_id].environment
```